getUserFeatureFlag/go.mod

```diff
@@ -6,9 +6,14 @@ require (
 	github.com/Real-Dev-Squad/feature-flag-backend v0.0.0-20230617074203-11973fea328a
 	github.com/aws/aws-lambda-go v1.41.0
 	github.com/aws/aws-sdk-go v1.44.284
+	github.com/google/uuid v1.3.0
```

Comment:
font we have inbuilt uuid inside crypto library?

Reply:
I thin we don't have that
https://pkg.go.dev/crypto#pkg-functions